docs/reference/cat/indices.asciidoc

```diff
@@ -75,6 +75,14 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=help]
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=include-unloaded-segments]
 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=local]
++
+--
+`local`::
+(Optional, boolean) If true, the request retrieves information from the local node
+only. Defaults to false, which means information is retrieved from the master node.
+deprecated::[8.0,This parameter does not cause this API to act locally. It will be
+removed in version 8.0.]
+--
```

Comment:
@boicehuang @danhermann You can simply remove the `include` statement. Since this parameter is currently no-op, we can just expand the deprecation message. Note you'll need to update the deprecation release based on where we backport this.
```suggestion
`local`::
(Optional, boolean)
+
deprecated::[7.10.0,"This parameter does not affect the request. It will be removed in version 8.0.0."]
```

Reply:
Ah, thank you, that makes sense. I see now that each include specifies a single tag from the file.